### DISCUSSION TITLE
함께 읽을 책 제안합니다

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [ThinkAboutSoftware/AcademicConference#1] [General]
어떤 주제로 대화를 나누면 좋을지 고민을 하다가 문득 어떤 개발자로 살아가면 좋을지에 대해서 서로 대화를 나눠보면 어떨까란 생각이 들었습니다
그래서 찾던 중에 [소프트웨어 장인](http://m.yes24.com/goods/detail/20461940)을 발견 했습니다😄
굳이 위 책이 아니어도 **진정한 개발자가 되기 위한 가이드**를 제시해줄 수 있는 책을 선정해서 진행하면 어떨까 의견 남깁니다 ㅎㅎ

**Comment by Joseph-Cha** [comment]
그리고 와이프 허락을 받았습니다 ㅎㅎㅎ(마음을 비우고 담담하게 이야기를 하니 오히려 별 의심없이 허락하더군요) 
다만 저희가 6월말 또는 7월 첫째주에 이사가 잡혀 있어서 일정을 조금 조절할 수 있을까요?ㅠ

**Reply by jongfeel** [reply]
소프트웨어 장인 매우 좋은 책입니다.

다른 멤버들도 좋다고 하면 저는 적극적으로 추천하고 싶네요.

그리고 미리 제안 주셔서 감사합니다.

**Reply by jongfeel** [reply]
7월이 되면 여름 성수기 시즌이라 가격이 좀 비싸집니다.
저는 상관 없긴 한데 같이 참여하는 멤버분의 부담을 덜어주기 위해 
6월 18일 혹은 6월 25일로 할까 생각 중이에요.

**Reply by Joseph-Cha** [reply]
네 감사합니다😊

**Reply by hi-hyein** [reply]
허락 축하드립니다~~~!!! 🎉 🎉 🎉 🎉 🎉 
소프트웨어 장인책 멘토링하면서 많이 들었었는데 
저도 이 기회에 읽으면 아주 좋을 것 같아요!

**Reply by yoonucho** [reply]
헐! 저는 요기를 이제 봤네요 

소프트웨어장인 책 좋습니다! 
2년전에 보긴 했는데 그때 본거와 또다른 깨달음을 얻을 수 있을거 같습니다!

**Reply by jongfeel** [reply]
아마 그냥 봤었을 때 하고 토론하면서 다시 보는거하고 많이 다른 느낌이실 거에요
